src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/breadcrumb.html.twig

```diff
@@ -28,25 +28,30 @@
     <nav>
       <ol class="breadcrumb">
         {% for category in currentCategoryView.breadcrumb_tree %}
+          {% set isRootCategory = (category.id_parent == 0) %}
           <li class="breadcrumb-item">
-            {% if category.id_parent == 0 and isSingleShopContext %}
+            {% if isRootCategory and isSingleShopContext %}
               {% if category.id_category == currentCategoryView.id %}
                 <span>{{ category.name }}</span>
-                <a href="{{ path('admin_categories_edit', {'categoryId': category.id_category}) }}">
-                  <i style="font-size: 16px;" class="material-icons">edit</i>
-                  {{ 'Edit'|trans({}, 'Admin.Actions') }}
-                </a>
+                  {% if not isRootCategory %}
```

Comment:
I'm not sure but it seems like this will never happen since we're already in a block where `isRootCategory and isSingleShopContext` The category is necessarily a root category so this link block can be removed I guess.